PointPrim.cpp

```diff
@@ -27,14 +27,6 @@
 
 BOOL __fastcall Interpreter::primitiveMakePoint(CompiledMethod&, unsigned argCount)
 {
-#ifdef TIMEDEXPIRY
```

Comment:
I see you found the bit in primitiveMakePoint. Perhaps you'd moved that already and I missed it.